src/pages/steps/PlaceStep.tsx

```diff
@@ -79,12 +82,14 @@ const PlaceStep = <TFormData extends FormDataUnion>({
 }: PlaceStepProps<TFormData>) => {
   const { t, i18n } = useTranslation();
   const [searchInput, setSearchInput] = useState('');
+  const [prefillPlaceName, setPrefillPlaceName] = useState('');
+  const [isPlaceAddModalVisible, setIsPlaceAddModalVisible] = useState(false);
 
   const useGetPlacesQuery = useGetPlacesByQuery(
     {
       name: searchInput,
       terms,
-      zip,
+      zip: municipality.zip,
```

Comment:
```suggestion
      zip: municipality?.zip,
```

Since it can be undefined